docker-compose.yml

```diff
@@ -162,16 +163,15 @@ services:
             ip link set vni10 up && \
             ip link set br10 up && \
 
-            #ip link add br20 type bridge && \
-            #ip link set br20 master blue addrgenmode none && \
-            #ip link set br20 addr aa:bb:cc:00:00:23 && \
-            #ip addr add 20.20.20.3/24 dev br20 && \
-            #ip link add vni20 type vxlan local 10.0.0.4 dstport 4789 id 20 nolearning && \
-            #ip link set vni20 master br20 addrgenmode none && \
-            #ip link set vni20 type bridge_slave neigh_suppress on learning off && \
-            #ip link set vni20 up && \
-            #ip link set br20 up && \
-
+            ip link add br20 type bridge && \
+            ip link set br20 master blue addrgenmode none && \
+            ip link set br20 addr aa:bb:cc:00:00:23 && \
+            ip addr add 20.20.20.3/24 dev br20 && \
```

Comment:
Same here. What is the role of this 20.20.20.3/24 IP address ?

Reply:
Below is what intended for the above configs without any hosts, please review the latest changes.
    # L2 VXLAN - VLAN10 stretched with VNI10 to leaf2 from leaf1 (ping 10.10.10.3)
    # L3VXLAN Asymmetric IRB - VLAN20 stretched with VNI20 to leaf2 from leaf1 (ping 20.20.20.3) on blue VRF
    # L3VXLAN Symmetric IRB - L3VNI 100 is used for borderleaf connectivity to reach the internet